tests/vmi_configuration_test.go

```diff
@@ -2233,7 +2233,7 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				// disk[1]:  Block, no user-input, cache=none
 				libvmi.WithPersistentVolumeClaim("block-pvc", dataVolume.Name),
 				// disk[2]: File, not-sparsed, no user-input, cache=none
-				libvmi.WithPersistentVolumeClaim("hostpath-pvc", tests.DiskAlpineHostPath),
+				libvmi.WithPersistentVolumeClaim("hostpath-pvc", "disk-alpine-host-path"),
```

Comment:
It seems that the usage is not isolated only to storage tests. Having a common variable makes it easier for future change. I doubt there will be any but please amend the reasoning.

Reply:
Yes but as far as I can see I could use any path here - I could of gone with `"vmi-config-host-path" ` so there is no need for the constant here.

Reply:
The argument refers to PVC's name. I am not sure I understand...

Reply:
/hold
in order to let you consider my opinion. Feel free to unhold.

Reply:
The name passsed to `WithPersistentVolumeClaim` is dependent on the PVC that is created in L2228.

> libstorage.CreateHostPathPVC("alpine-host-path", testsuite.GetTestNamespace(nil), "1Gi")

The code behind `CreateHostPathPVC` creates a PVC with name format `disk-%s`, which results in the PVC being named `disk-alpine-host-path`.

So @brianmcarey is right, that the name has no deeper meaning here. But I would still introduce a const, to sync both the call to `libstorage.CreateHostPathPVC` and `libvmi.WithPersistentVolumeClaim`.

Reply:
Something like the following : 
https://github.com/kubevirt/kubevirt/compare/fb92eb201b29b7143119804c55bc2658aba10fe1..882a21136682a51092cba919e46bfa59e4242540

Reply:
@dankenigsberg I agree that it probably could be a unit test but I think sig-compute have some ongoing work in identifying e2e tests that can be converted to unit tests so I will leave that to sig-compute for now.

Reply:
> The name passsed to WithPersistentVolumeClaim is dependent on the PVC that is created in L2228.

Thanks for explaining this. It should be fixed. The `disk-` prefix is an implementation detail of `libstorage.CreateHostPathPVC`; we should not have `disk-` repeated outside of it. How about having `libstorage.CreateHostPathPVC` return the generated path?

/hold cancel